### PR TITLE
Emily_Young_HW1

### PR DESCRIPTION
**Pull request by emilyskyoung**
Reflection on Homework

Overall, I found learning how to use Markdown relatively straight forward. I utilized the RMarkdown Cheat Sheet and its directions were easy to follow. I did find that I would occasionally forget to place my indications for stylization on both sides of the text, but I usually was able to find and solve this problem quickly.  I completed the homework through a trial and error process. I would find problems with the code not creating what I wanted it to in the preview document and would then have to go back and fix those problems before previewing the document again to make sure my code did what I wanted it to. I assume that this trial and error process will happen a lot as I complete the other homework assignments. 

One thing that I did have trouble with was figuring out how to use GitHub correctly. Originally, I forgot to fork the repository before editing the ReadMe file, which led to a multitude of problems trying to upload my final code. Eventually, with help from a friend, I decided to fork the repository, create an entirely new project in R, and then copy and paste my original code over into the new project. That is why, while I did commit my original code multiple times, the one that I am submitting only has a few commits. I will try to become more familiar with the process of using GitHub in the future so that this problem doesn't happen again. 

Lastly, I ran into trouble when trying to edit my code and then push it back to the repository. Because I had created these two different repositories - one that was forked from our class page and one that wasn't - it became confusing to figure out which one I was working in. However, after working on it for awhile, I asked for help on our class GitHub page and received an answer almost immediately! While I was at first a little embarrassed to ask for help so publicly, I think that this initial first try using our GitHub help page, and the speed with which I got a response, will encourage me to keep asking for help in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
-# hw01
+# Emily Young - HW01
```
Comment (lindsaykkk) [inline README.md:1]: This is a really detailed intro of yourself and I really like it!

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+4. Despite my inability to dance, sing, or act, I really love musical theater
+and frequently listen to Broadway cast recordings while studying. Unfortunately, my friends
+are not avid fans of theater - so if you ever want to talk about musicals I would be 
+*more than happy* to do so.
+5. I have a giant dog named Bruder who lives with my parents in London.
+He is big enough to fit into human-sized clothes and makes an excellent pillow/rug.
+
+***
+
+If you've read this entire biography I think you deserve a picture of Bruder so here you go:
```
Comment (lindsaykkk) [inline README.md:43]: It's a great picture! You could also try to insert a link the next time.

File: README.md (hunk 1 above)
Comment (lindsaykkk) [inline README.md:1]: This is a really detailed intro of yourself and I really like it! It's comfortable to read when you make some important terms in bold throughout the bio.

File: README.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+
+In terms of academic interests, I am really interested in the inequities present in our education 
+system and in studying how we can craft policies to remedy them.
+After college, I hope to either become a teacher or to work in the field of education 
+policy. Because of this, I plan to take as many education
+classes as I possibly can while I am here. 
+
+But enough about academics, here's a little bit about me personally: 
+
+1. I am from New York City and, like most New Yorkers, I am very proud of this fact.
```
Comment (lindsaykkk) [inline README.md:19]: This is a great list with a lot of interesting information!

**Review by lindsaykkk — COMMENTED** [review]
Overall, I think this is a really great self-intro and a good utilization of RMarkdown! Good work!

File: README.md (hunk 1 above)
Comment (sharanyabashyam) [inline README.md:1]: Hi Emily, reading your introduction, was very simple and easy. You gave a very detailed list, and highlighted some words in the middle which made it all the more fun to read!

File: README.md [hunk 4]
```diff
@@ -1 +1,45 @@
...
+and frequently listen to Broadway cast recordings while studying. Unfortunately, my friends
+are not avid fans of theater - so if you ever want to talk about musicals I would be 
+*more than happy* to do so.
+5. I have a giant dog named Bruder who lives with my parents in London.
+He is big enough to fit into human-sized clothes and makes an excellent pillow/rug.
+
+***
+
+If you've read this entire biography I think you deserve a picture of Bruder so here you go:
+![](IMG_4301.jpeg)
```
Comment (sharanyabashyam) [inline README.md:44]: Not related to the review but I love Bruder

File: README.md [hunk 5]
```diff
@@ -1 +1,45 @@
...
+
```
Comment (sharanyabashyam) [inline README.md:45]: I think you pretty much used most of the features of RMarkdown that we were asked to except inserting a link. Maybe you could add that next time. Overall, I enjoyed reading this

File: README.md (hunk 2 above)
Comment (ming-cui) [inline README.md:43]: Hi Emily, maybe you can insert the link here. Currently, the picture doesn't show up in your readme file.

File: README.md [hunk 6]
```diff
@@ -1 +1,45 @@
...
+policy. Because of this, I plan to take as many education
+classes as I possibly can while I am here. 
+
+But enough about academics, here's a little bit about me personally: 
+
+1. I am from New York City and, like most New Yorkers, I am very proud of this fact. 
+However, when I moved to Chicago for college my parents decided to move to London, England.
+While I'm definitely not a Londoner (a fact my 18 year old
+brother loves rubbing in my face) I love visiting the city and my family during school breaks.  
+2. For extracurricular activities:
```
Comment (ming-cui) [inline README.md:23]: Would a space line here make your code more readable?

File: README.md [hunk 7]
```diff
@@ -1 +1,45 @@
-# hw01
+# Emily Young - HW01  
+###### *Below I will practice using RMarkdown by writing a biography about myself*
```
Comment (ming-cui) [inline README.md:2]: I like this comment line.

**Review by ming-cui — COMMENTED** [review]
A general suggestion. Using more headings may increase the readability of your code. :)

File: README.md [hunk 8]
```diff
@@ -1 +1,45 @@
...
+While I'm definitely not a Londoner (a fact my 18 year old
+brother loves rubbing in my face) I love visiting the city and my family during school breaks.  
+2. For extracurricular activities:
+    + I am a teacher's assistant for a 3rd grade math class at a local Chicago Public School.
+    + I help run a Model UN conference for high school students to teach them debate skills. 
+  This year we are simulating the woman's suffrage movement and as an **avid** feminist 
+  I cannot wait to see how the kids tackle this challenge.  
+    + I am a project leader for UChicago's student-run think tank, the Paul
+  Douglas Institute.  
+3. I can't drink coffee because I find the taste horrible. Instead, I have a
```
Comment (bensoltoff) [inline README.md:30]: I could never enjoy tea, but recently started drinking some herbal teas to cut down on my coffee intake. So far I've enjoyed the peppermint and autumn harvest teas from Trader Joes. And I am quite excited for the candy cane tea to come back in a couple of months

File: README.md [hunk 9]
```diff
@@ -1 +1,45 @@
...
+2. For extracurricular activities:
+    + I am a teacher's assistant for a 3rd grade math class at a local Chicago Public School.
+    + I help run a Model UN conference for high school students to teach them debate skills. 
+  This year we are simulating the woman's suffrage movement and as an **avid** feminist 
+  I cannot wait to see how the kids tackle this challenge.  
+    + I am a project leader for UChicago's student-run think tank, the Paul
+  Douglas Institute.  
+3. I can't drink coffee because I find the taste horrible. Instead, I have a 
+**very large** tea collection which I *love* sharing with friends. If you ever need 
+recommendations for the best type of tea to drink (my answer is usually Earl Grey)
```
Comment (bensoltoff) [inline README.md:32]: Not sure I can let that pass without a Picard reference (his favorite beverage)

File: README.md (hunk 8 above)
Comment (bensoltoff) [inline README.md:30]: Note you don't have to manually number each item. Markdown does that for you when you render the document.

```
1. A list
1. Another item
1. Just filling up space
```

Renders as

1. A list
1. Another item
1. Just filling up space

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Reproducibility**         |           |        x      |            |

* [See these instructions if you forget to fork in the future](https://cfss.uchicago.edu/notes/common-git-problems/#accidentally-cloned-from-the-master-not-the-fork). It allows you to change the URL you are pushing to, rather than having to delete the local copy and start fresh

**Pull request closed without merging**